OrchardCore.Commerce.Tests/CurrencyTests.cs

```diff
@@ -18,7 +18,7 @@ public static CurrencyTheoryData TestData
                 { CanadianDollar, 1234.56m, "$1,234.56" },
                 { SwissFranc, 1234.56m, "CHF 1’234.56" },
                 { ChineseYuan, 1234.56m, "¥1,234.56" },
-                { new Currency("My FOO", "My FOO", "f", "FOO"), 1234.56m, "(FOO) 1,234.56" }
+                { new Currency("My FOO", "My FOO", "f", "FOO"), 1234.56m, $"(FOO) {1234.56m.ToString("N")}" }
```

Comment:
Based on this : 

```
        public string ToString(decimal amount)
            => Culture is null
            ? "(" + CurrencyIsoCode + ") " + amount.ToString("N" + DecimalPlaces)
            : amount.ToString("C" + DecimalPlaces, Culture);
```

I think it is appropriate since we do a format of the amount there too.